test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -0,0 +1,703 @@
+    accessor: 'score'
+    },
+    {Header: 'Performance Analysis',
+    accessor: 'analysis'
+    },
+];
+	
+const legendRows = [{
+    colorName: 'Green',
+    color: '#2dc937',
+    score: ">98%",
+    analysis: "No Regression"
+    }, {
+    colorName: 'Yellow',
+    color: '#F0F755',
+    score: "91-98%",
+    analysis: "Possible Regression"
+    }, {
+    colorName: 'Red',
+    color: '#cc3232',
+    score: "<98%",
+    analysis: "Regression"
+    },
+   {colorName: 'Grey',
+    color: 'grey',
+    score: "N/A",
+    analysis: "No Data"
+   	},
+   {colorName: 'Off-White',
+    color: '#ffdbac',
+    score: "No Percentage",
+    analysis: "Only test/baseline data available"
+   	}];
+
+// Overlay for the date picker component
+function CustomOverlay({ classNames, selectedDay, children, ...props }) {
+    return (
+        <div
+        className={classNames.overlayWrapper}
+        style={{ marginLeft: -100 }}
+        {...props}
+    >
+        <div className={classNames.overlay}>
+        <p>
+            {selectedDay
+            ? `Currently Chosen JDK Date: ${selectedDay.toLocaleDateString()}`
+            : 'Choose JDK Date'}
+        </p>
+        {children}
+        </div>
+    </div>
+);}
+
+CustomOverlay.propTypes = {
+    classNames: PropTypes.object.isRequired,
+    selectedDay: PropTypes.instanceOf(Date),
+    children: PropTypes.node.isRequired,
+};
+
+export default class TabularView extends Component {
+    constructor(props) {
+    super(props);
+    this.handleDayChange = this.handleDayChange.bind(this);
+    this.state = {testData: [], columns : [], originalColumns: [], platforms: [], baselineData: [], consolidatedData: [], platformFilter: [], colorFilter: "all", benchmarkFilter: []};
+    }
+ 
+    componentDidMount() {
+        const current_url = window.location.href;	
+        const path = current_url.split("?")[1];
+        if (path) {
+            const stateVars = path.split("&");
+            // Populate options from URL
+            for (const item in stateVars) {
+                const key = stateVars[item].split("=")[0];
+                const value = stateVars[item].split("=")[1];
+                this.state[key] = value;
+            }
+        }
+        this.initializeJdk();
+        this.showData('test');
+        this.showData('baseline');
+     }
+
+    // Set default values for choosing the JDK including date, version, type and sdk resource if URL has missing information
+    initializeJdk () {
+        const date = (new Date().getDate()).toString();
+        const month = (new Date().getMonth() + 1).toString(); //Current Month
+        const year = (new Date().getFullYear()).toString(); //Current Year
+
+        const jdkDate = year + ((month.length < 2) ? "0" + month : month) + ((date.length < 2) ? "0" + date : date)
+        
+        !('baselineJdkDate' in this.state) && (this.state.baselineJdkDate = jdkDate);
+        !('baselineJdkVersion' in this.state) && (this.state.baselineJdkVersion = 'O8');
```

Comment:
Again! These are our assumptions so we should list them all out along with an example.